source/utils/getUpdatedOffsetForIndex.js

```diff
@@ -19,12 +19,15 @@ export default function getUpdatedOffsetForIndex ({
 }) {
   const maxOffset = cellOffset
   const minOffset = maxOffset - containerSize + cellSize
+  const midOffset = maxOffset - (containerSize + cellSize) / 2
```

Comment:
No need to calculate this here. Just return it inline down below, in the `'center'` case.

min and max offsets are only calculated here because they're used by multiple cases below.